_posts/2022-10-19-trino-summit-2022-teaser-2.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Trino Summit 2022: Federating humans and data"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
+---
+
+Trino has long been the de facto standard to querying large data sets over your
+cloud or on-prem storage, also known as data lakes. This Trino Summit's theme 
+instead will showcase Trino's other claim to fame: query federation. In essence,
+Trino is a query engine providing an access point that exposes ANSI SQL across 
+[multiple data sources]({{site.url}}/docs/current/connector.html).
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
+is free and takes place on November 10th. Check out the details below!
```

Comment:
do we need to add "in San Francisco" ?

Reply:
Also .. should we remove "Check out the details below!" .. there is already a read more link

Reply:
Fair, I just wanted it to be clear that we'll provide more details about how to attend if someone was lazy but it should be obvious to read on.